FRENCH/src/ch06-02-match.md

```diff
@@ -0,0 +1,673 @@
+possibility in order for the code to be valid. Especially in the case of
+`Option<T>`, when Rust prevents us from forgetting to explicitly handle the
+`None` case, it protects us from assuming that we have a value when we might
+have null, thus making the billion-dollar mistake discussed earlier.
+-->
+
+Rust sait que nous n'avons pas couvert toutes les possibilités et sait même quel
+motif nous avons oublié ! Les `match` de Rust sont *exhaustifs* : nous devons
+traiter toutes les possibilités afin que le code soit valide. Dans notre cas de
+`Option<T>`, quand Rust nous empêche d'oublier de gérer explicitement le cas de
+`None`, il nous protège d'une situation où nous supposons que nous avons une
+valeur alors que nous pourrions avoir null, comme l'erreur au milliards de
+dollars que nous avons vu précédemment.
+
+<!--
+### The `_` Placeholder
+-->
+
+### Le caractère réservé `_`
+
+<!--
+Rust also has a pattern we can use when we don’t want to list all possible
+values. For example, a `u8` can have valid values of 0 through 255. If we only
+care about the values 1, 3, 5, and 7, we don’t want to have to list out 0, 2,
+4, 6, 8, 9 all the way up to 255. Fortunately, we don’t have to: we can use the
+special pattern `_` instead:
+-->
+
+Rust a aussi un motif que nous pouvons utliser quand nous ne voulons pas traiter
+toutes les valeurs possibles. Par exemple, un `u8` peut être valide entre 0
+et 255. Si nous n'avons besoin que des valeurs 1, 3, 5 et 7, nous ne voulons pas
+traiter 0, 2, 4, 6, 7, 8, 9 et ainsi de suite jusqu'à 255. Heureusement, nous
+n'avons pas à le faire : nous pouvons utiliser le motif spécial `_` :
+
+<!--
+```rust
+let some_u8_value = 0u8;
+match some_u8_value {
+    1 => println!("one"),
+    3 => println!("three"),
+    5 => println!("five"),
+    7 => println!("seven"),
+    _ => (),
+}
+```
+-->
+
+```rust
+let une_valeur_u8 = 0u8;
+match une_valeur_u8 {
+    1 => println!("un"),
+    3 => println!("trois"),
+    5 => println!("cinq"),
+    7 => println!("sept"),
+    _ => (),
+}
+```
+
+<!--
+The `_` pattern will match any value. By putting it after our other arms, the
+`_` will match all the possible cases that aren’t specified before it. The `()`
+is just the unit value, so nothing will happen in the `_` case. As a result, we
+can say that we want to do nothing for all the possible values that we don’t
+list before the `_` placeholder.
+-->
+
+Le motif `_` va correspondre à toutes les valeurs. En l'ajoutant à la fin de nos
+branches, le `_` va correspondre à tous les autres cas qui ne sont pas listés
+avant. Au final, nous avons pu dire que nous ne voulons rien faire pour les
+valeurs possibles que nous ne traitons pas avant le caractère réservé `_`.
+
+<!--
+However, the `match` expression can be a bit wordy in a situation in which we
+care about only *one* of the cases. For this situation, Rust provides `if let`.
+-->
+
+Cependant, l'expression `match` peut être un peu verbeuse dans une situation où
```

Comment:
Even if `verbeux` does exist in French, I'm not sure that non-english speakers can guess its meaning if they don't know the word without doing research, I don't think we want that to happen

Reply:
Okay, what do you think of `lourde` or ` pompeuse`, in this case ?

Reply:
I'd say `lourde` or `inutilement longue` could fit well here